src/endpoints/http/collections/export/index.js

```diff
@@ -0,0 +1,143 @@
+import parsers from './lib/parser';
+import archive from './lib/archive';
+import UnsupportedMediaError from '../../../../Errors/UnsupportedMediaError';
+
+const GB = 1073741824; // this will be configurable through fh-config
```

Comment:
Can you add this functionality as part of this PR?

I do the same with the permission stuff https://github.com/feedhenry/fh-dataman/blob/master/src/middleware/route-authorize/permission.js#L13

Just add the value to `config/dev.json`